# Logfile created on Thu Feb 03 22:34:07 -0500 2011 by /
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::ENETUNREACH: Network is unreachable - connect(2)>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: No active servers>
#<Errno::ENETUNREACH: Network is unreachable - connect(2)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::ENETUNREACH: Network is unreachable - connect(2)>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: No active servers>
#<Errno::ENETUNREACH: Network is unreachable - connect(2)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Workling::WorklingConnectionError: MemCache::MemCacheError - IO timeout>
#<MemCache::MemCacheError: IO timeout>
#<Timeout::Error: IO timeout>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: No active servers>
#<Errno::ECONNREFUSED: Connection refused - connect(2)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Base>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Discovery>
#<Gem::LoadError: Could not find RubyGem fiveruns-memcache-client (>= 0)
>
#<NameError: uninitialized constant Rails::Info>
#<SystemExit: exit>
#<Gem::LoadError: Could not find RubyGem hoptoad_notifier ()
>
#<NameError: uninitialized constant Workling::Base>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Base>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Discovery>
#<Gem::LoadError: Could not find RubyGem fiveruns-memcache-client (>= 0)
>
#<NameError: uninitialized constant Rails::Info>
#<SystemExit: exit>
#<Gem::LoadError: Could not find RubyGem hoptoad_notifier ()
>
#<NameError: uninitialized constant Workling::Base>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Base>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Discovery>
#<Gem::LoadError: Could not find RubyGem fiveruns-memcache-client (>= 0)
>
#<NameError: uninitialized constant Rails::Info>
#<SystemExit: exit>
#<Gem::LoadError: Could not find RubyGem hoptoad_notifier ()
>
#<NameError: uninitialized constant Workling::Base>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Base>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Discovery>
#<Gem::LoadError: Could not find RubyGem fiveruns-memcache-client (>= 0)
>
#<NameError: uninitialized constant Rails::Info>
#<SystemExit: exit>
#<Gem::LoadError: Could not find RubyGem hoptoad_notifier ()
>
#<NameError: uninitialized constant Workling::Base>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Mysql::Error: Too many connections>
#<Mysql::Error: Too many connections>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: No active servers>
#<Errno::ECONNREFUSED: Connection refused - connect(2)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
#<MissingSourceFile: no such file to load -- spawn>
#<NameError: uninitialized constant Workling::Remote>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: No active servers>
#<Errno::ECONNREFUSED: Connection refused - connect(2)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
#<MissingSourceFile: no such file to load -- spawn>
#<NameError: uninitialized constant Workling::Remote>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: IO timeout>
#<Timeout::Error: IO timeout>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
#<MissingSourceFile: no such file to load -- spawn>
#<NameError: uninitialized constant Workling::Remote>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<MissingSourceFile: no such file to load -- xmlscan/scanner>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<MissingSourceFile: no such file to load -- jira4r>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<EOFError: end of file reached>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<EOFError: end of file reached>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: No active servers>
#<Errno::ECONNREFUSED: Connection refused - connect(2)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
#<MissingSourceFile: no such file to load -- spawn>
#<NameError: uninitialized constant Workling::Remote>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Workling::WorklingConnectionError: MemCache::MemCacheError - IO timeout>
#<MemCache::MemCacheError: IO timeout>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Workling::WorklingConnectionError: MemCache::MemCacheError - IO timeout>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: IO timeout>
#<Timeout::Error: IO timeout>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
#<MissingSourceFile: no such file to load -- spawn>
#<NameError: uninitialized constant Workling::Remote>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Mysql::Error: Can't connect to MySQL server on '10.4.0.30' (111)>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<NameError: uninitialized constant Workling::Remote>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Errno::EAGAIN: Resource temporarily unavailable>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Remote>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: IO timeout>
#<Timeout::Error: IO timeout>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
#<MissingSourceFile: no such file to load -- spawn>
#<NameError: uninitialized constant Workling::Remote>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<NameError: uninitialized constant Workling::Routing>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<MissingSourceFile: no such file to load -- spawn>
#<NameError: uninitialized constant Workling::Remote>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<MemCache::MemCacheError: IO timeout>
#<Timeout::Error: IO timeout>
#<Errno::EAGAIN: Resource temporarily unavailable>
#<Workling::QueueserverNotFoundError: config/workling.yml configured to connect to queue server on 10.4.0.30:22122 for this environment. could not connect to queue server on this host:port. for starling users: pass starling the port with -p flag when starting it. If you don't want to use Starling, then explicitly set Workling::Remote.dispatcher (see README for an example)>
#<NameError: uninitialized constant Workling::Remote::Invokers::ThreadedPoller>
#<NameError: uninitialized constant Workling::Remote::Invokers>
#<NameError: uninitialized constant Workling::Routing::ClassAndMethodRouting>
#<NameError: uninitialized constant Workling::Routing>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<MissingSourceFile: no such file to load -- encoding/character/utf-8>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<MissingSourceFile: no such file to load -- rails>
#<ArgumentError: couldn't find HOME environment -- expanding `~'>
#<MissingSourceFile: no such file to load -- commonwatir>
#<MissingSourceFile: no such file to load -- encoding/character/utf-8>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<MissingSourceFile: no such file to load -- encoding/character/utf-8>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<MissingSourceFile: no such file to load -- rails>
#<ArgumentError: couldn't find HOME environment -- expanding `~'>
#<MissingSourceFile: no such file to load -- commonwatir>
#<MissingSourceFile: no such file to load -- encoding/character/utf-8>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<MissingSourceFile: no such file to load -- encoding/character/utf-8>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<MissingSourceFile: no such file to load -- rails>
#<ArgumentError: couldn't find HOME environment -- expanding `~'>
#<MissingSourceFile: no such file to load -- commonwatir>
#<MissingSourceFile: no such file to load -- encoding/character/utf-8>
*** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
#<MissingSourceFile: no such file to load -- encoding/character/utf-8>
*** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<NoMemoryError: failed to allocate memory>
#<SystemStackError: stack level too deep>
#<fatal: exception reentered>
#<MissingSourceFile: no such file to load -- rails>
#<ArgumentError: couldn't find HOME environment -- expanding `~'>
#<MissingSourceFile: no such file to load -- commonwatir>
#<MissingSourceFile: no such file to load -- encoding/character/utf-8>
